src/main/java/com/hubspot/smtp/client/SmtpSessionConfig.java

```diff
@@ -1,25 +1,32 @@
 package com.hubspot.smtp.client;
 
 import java.net.InetSocketAddress;
+import java.time.Duration;
+import java.util.Optional;
 
-public class SmtpSessionConfig {
-  private final InetSocketAddress remoteAddress;
-  private final InetSocketAddress localAddress;
+import org.immutables.value.Value.Default;
+import org.immutables.value.Value.Immutable;
```

Comment:
Now that we're using Immutables, can you add the following to `.travis.yaml` to fix travis CI?

```yaml
script: mvn -B -q clean install
```

The important thing here is `clean` to fix the generated sources bug